packages/story-editor/src/components/mediaRecording/playPauseButton.js

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
This file appears to be a copy and paste from the video element. Is there anyway to move this component to be spared?

Reply:
We also have a play / pause button here 
https://github.com/GoogleForCreators/web-stories-wp/blob/d8b28a203a9874c890d99aa1ad24935a48a6a8b1/packages/story-editor/src/components/panels/shared/media/audioPlayer.js#L173-L181

I wonder these could all use the same compontent.

Reply:
> I am noticing the recording icon is still showing in the tabs and I am seeing the light on the web cam still on, long after recording is finished.

The component used for the video element is much more complex, it can't be reused here

As for the play button itself, maybe something for the future to move that to the design system or something.